packages/office-ui-fabric-react/src/components/Facepile/Facepile.base.tsx

```diff
@@ -58,7 +58,7 @@ export class FacepileBase extends BaseComponent<IFacepileProps, {}> {
         {this.onRenderAriaDescription()}
         <div className={_classNames.itemContainer}>
           {showAddButton ? this._getAddNewElement() : null}
-          <ul className={_classNames.members} role="listbox" aria-label={ariaLabel}>
```

Comment:
Why is this role being removed? It's needed for ATs to understand groups of Facepiles when they are in a list of people right? ARIA says we can use HTML `<select>` instead but it doesn't support images.

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role

Reply:
The link you pasted says the following.

> The listbox role is used to identify an element that creates a list from which a user may select one or more static items

I think the Facepile is just a "list" of people, and not a picker of any sort, where the user can **select** one.

Currently it will announce it as a "group"/"list" which might make more sense. Especially since the list can be a mix of clickable and non-clickable items.